theme/static/js/hs_file_browser.js

```diff
@@ -1080,19 +1080,14 @@ $(document).ready(function () {
 
     function onPaste() {
         var folderName = $("#fb-files-container li.ui-selected").children(".fb-file-name").text();
-        var targetPath = $("#hs-file-browser").attr("data-current-path");
-
-        if (folderName && folderName.lastIndexOf(".") == -1) {  // Makes sure the destination is a folder
```

Comment:
This IF clause was used to append the destination folder name to the target path when a user right clicks on the folder icon and selects 'Paste' from the menu. 

I believe this use case won't work correctly with the removal of this code.